### ISSUE TITLE
Headings containing a hyphen (-) don't get automatically linked

### ISSUE THREAD

**Issue by TimMoore** [lightbend/lightbend-markdown#2]
Contrast http://www.lagomframework.com/documentation/1.2.x/java/ReadSide.html#raw-stream-of-events

rendered as:

```html
<h2 id="Raw-Stream-of-Events"><a class="section-marker" href="#Raw-Stream-of-Events">§</a><a href="#raw-stream-of-events" name="raw-stream-of-events">Raw Stream of Events</a></h2>
```

with http://www.lagomframework.com/documentation/1.2.x/java/ReadSide.html#Read-side-design

```html
<h2 id="Read-side-design"><a class="section-marker" href="#Read-side-design">§</a>Read-side design</h2>
```

I can reproduce this locally by adding or removing a hyphen.

**Comment by TimMoore** [comment]
Actually looks like a bug in play-doc or possibly pegdown

**Comment by jroper** [comment]
Fixed in https://github.com/playframework/play-doc/pull/34.

**Comment by TimMoore** [comment]
Oh I jumped the gun closing... I guess we need to update the version of play-doc used.

**Comment by jroper** [comment]
Fixed and released as 1.5.1.
